languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -133322,7 +133322,20 @@ The accident victim died from her injuries.
                 <example correction="agent's|agent’s">Go to the travel <marker>agent`s</marker> office.</example>
             </rule>
         </rulegroup>
-
+        <rule id="CONFUSION_OF_WHEN_WHAT" name="confusion of when/what">
```

Comment:
This should probably have `default="temp_off"` so it's in the nightly first and not immediately active.